tests/integration/zome_call.rs

```diff
@@ -0,0 +1,87 @@
+use base64::prelude::*;
+use fixt::prelude::*;
+use holochain::fixt::DnaHashFixturator;
+use holochain_http_gateway::{
+    config::{AllowedFns, Configuration},
+    tracing::initialize_tracing_subscriber,
+};
+use reqwest::StatusCode;
+
+use crate::TestApp;
+
+#[tokio::test]
+async fn zome_call_uses_correct_route_parameters() {
+    initialize_tracing_subscriber("info");
+
+    let app = TestApp::spawn().await;
+
+    let dna_hash = fixt!(DnaHash);
```

Comment:
`fixt!` is okay, and it can be helpful for generating test data. I think for what we're doing here, it's probably not necessary to have a dependency on it. A DNA hash is just a byte array as far as we're concerned here. Doing `DnaHash::from_raw_36(vec![12; 36]);` will get the job done and be recognized as a `DnaHash` by the receiver.

When it comes to testing with a fixture hApp, we'll have a real DNA hash to use.

Reply:
I tried `DnaHash::from_raw_36` but I was encountering this error: `Holo Hash checksum validation failed`. I don't experience the same error with `fixt`

Reply:
Oh that's interesting. We have many tests where we do that but when we deserialize from string, the location bytes get checked. In that case, you can still use `from_raw_32`, that will calculate valid location bytes from the input.

Reply:
This thread is still open

Reply:
Oh my bad, I tried creating a DnaHash with `from_raw_32` in the tests too, but this yielded a panic from assertion `failed: invalid byte count for HoloHash [219, 32]`

Reply:
That's odd. I'd have to give that a try myself to understand why. This definitely should be possible to construct without having a valid keypair.